### ISSUE TITLE
Multi images for Calculation  content loss by vgg

### ISSUE THREAD

**Issue by happy-yang** [nathankong/TextureSynthesis#1]
Calculation lots of images content loss use vgg
    def get_texture_loss(self):
        total_loss = 0.0
        for layer in self.layer_weights.keys():
            layer_activations = self.model_layers[layer]
            layer_activations_shape = layer_activations.get_shape().as_list()
            assert len(layer_activations_shape) == 4 # (1, H, W, outputs)
            **assert layer_activations_shape[0] == 1, "Only supports 1 image at a time."**
            num_filters = layer_activations_shape[3] # N
            num_spatial_locations = layer_activations_shape[1] * layer_activations_shape[2] # M
            layer_gram_matrix = self._compute_gram_matrix(layer_activations, num_filters, num_spatial_locations)
            desired_gram_matrix = self.constraints[layer]

            total_loss += self.layer_weights[layer] * (1.0 / (4 * (num_filters**2) * (num_spatial_locations**2))) \
                          * tf.reduce_sum(tf.pow(desired_gram_matrix - layer_gram_matrix, 2))
        return total_loss

 **assert layer_activations_shape[0] == 1, "Only supports 1 image at a time."**
how could I word for a Batch images  instead of 1
